server/src/rest_service.py

```diff
@@ -63,7 +63,7 @@ def add_team(db_session, data):
         return True, "Successfully added Team {}".format(data["name"])
     except Exception as e:
         db_session.rollback()
-        return False, "Could not add Team {}; already exists".format(data["name"])
+        return False, "Could not add Team {}; already exists".format(e)
```

Comment:
Sure, but then we should remove the part of the string that says `already exists` and only display the error message.